Modules/_testcapimodule.c

```diff
@@ -4846,6 +4846,29 @@ sequence_setitem(PyObject *self, PyObject *args)
 }
 
 
+static PyObject *
+hasattr_string(PyObject *self, PyObject* args)
+{
+    PyObject* obj;
+    PyObject* attr_name;
+
+    if(!PyArg_UnpackTuple(args, "hasattr_string", 2, 2, &obj, &attr_name))
+        return NULL;
+
+    if(!PyUnicode_Check(attr_name))
+    {
+        PyErr_SetString(PyExc_TypeError, "attribute name must a be string");
+        return PyErr_Occurred();
+    }
+
+    const char *name_str = PyUnicode_AsUTF8(attr_name);
+    if(PyObject_HasAttrString(obj, name_str))
+        Py_RETURN_TRUE;
+    else
+        Py_RETURN_FALSE;
+}
```

Comment:
Sorry for the nit, but I don't think there's automatic tooling to check this.  Can you reformat this to comply with [PEP7](https://peps.python.org/pep-0007/).